### ISSUE TITLE
[low] Chrome + Talkback: Screen reader announce 'Download','Copy to clipboard','Copy link','Mark as unread','Delete'; twice.

### ISSUE THREAD

**Issue by SumitDiyora** [Expensify/App#9571]
## Action Performed:
1. Using JAWS + Chrome browser, open the URL staging.new.expensify.com/r/89343211
2. Select any attachment from the chat.
3. Right-click on the attachment or activate it by using hover.
4. Navigate to the elements of the newly opened popup and listen to the screen reader announcement.

## Expected Result:
Upon moving the keyboard focus on the popup contents like 'Download','Copy to clipboard','Copy link','Mark as unread','Delete', The screen reader should announce the label only once.

## Actual Result:
Screen reader announces 'Download','Copy to clipboard','Copy link','Mark as unread','Delete'' twice.

## Workaround:
Yes

## Area issue was found in:
Chat

## Failed WCAG checkpoints
1.3.1

## User impact:
Screen reader users hear the announcements twice which is annoying and confusing.

## Suggested resolution:
Ensure the 'Text field label' announced only once by the screen reader.
Refer to:
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

## Platform:
- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:*v1.1.78-8*
**Reproducible in staging?:*Yes*
**Reproducible in production?:*Yes*
**Issue reported by:*Sumit*

![screen reader announce twice](https://user-images.githubusercontent.com/98817464/175975246-fb74b81c-29a6-47b3-96e4-cd4041896490.jpg)
